cassandra/cassandra-docker-ready.sh

```diff
@@ -22,7 +22,7 @@
 HOSTIP=`cat /etc/hosts | grep $HOSTNAME | awk '{print $1}' | head -n 1`
 
 # Get the status of this machine from the Cassandra nodetool
-STATUS=`nodetool status | grep $HOSTIP | awk '{print $1}'`
+STATUS=`nodetool status | grep " $HOSTIP " | awk '{print $1}'`
```

Comment:
I think it might work better with `-w` instead of spaces.

Example input:

```
192.168.0.1
192.168.0.10
```

Commands:

```bash
$ grep 192.168.0.1 /tmp/test.txt 
192.168.0.1
192.168.0.10
$ grep -w 192.168.0.1 /tmp/test.txt 
192.168.0.1
$ grep " 192.168.0.1 " /tmp/test.txt
$
```

Also, it would perhaps be better to properly wrap the var name like `${HOSTIP}`. Some versions of bash might use the env var `HOST` + the string `ip` if the var `HOSTIP` isn't set.

Reply:
This works with our script even if it doesn't work with your example :)

Using -w also works here, but I don't know if its necessarily better or not.